File: JS Version/project7.js
//By listing the first six prime numbers: 2, 3, 5, 7, 11, and 13, we can see that the 6th prime is 13.
//What is the 10 001st prime number?

function primeNumberFinder(){
    var count = 0
    for (var i = 2; true; i++){
        if (PrimeRibFinder(i)){
            count += 1
            if (count == 10001){
                return i
            }
        }
    }
}

function PrimeRibFinder(number){
    for (var i = 2; i <= Math.sqrt(number); i++){
        if (number % i == 0){
            return false  //if using return true , #winner, if using return false, divisble by some other shit so its not prime fuck you
        }
    }
    return true
}

console.log(primeNumberFinder())

File: project7.cpp
// By listing the first six prime numbers: 2, 3, 5, 7, 11, and 13, we can see that the 6th prime is 13.

// What is the 10 001st prime number?

#include <iostream>
#include <cmath>

int primeFinder(){
    int temp = 0;
    for (int i = 2; true; i++){
        for (int g = 2; g <= sqrt(i); g++){
            if (i % g != 0){
                temp += 1;
                if (temp = 10001){
                    return i;
                }
            }
        }
    }
}



int main(){
    std::cout << primeFinder() << std::endl;
}